app/config/sources.inc.php

```diff
@@ -437,8 +439,18 @@
             'it', 'kab', 'ko', 'pl', 'pt-BR', 'sq', 'sv-SE', 'uk', 'zh-TW',
         ],
     ],
+    'mozorg/internet-health/decentralization.lang' => [
+        'deadline' => '2017-05-24',
+        'flags'    => [
+            'opt-in' => ['all'],
+        ],
+        'priority'          => 3,
```

Comment:
Why are you giving priority 3 to this while all the other have priority 1? Also note that priority 3 is the default for mozilla.org, you wouldn't need to set it explicitly.